quinn-h3/Cargo.toml

```diff
@@ -17,6 +17,9 @@ edition = "2018"
 [package.metadata.docs.rs]
 all-features = true
 
+[features]
+interop_test_accessors = []
```

Comment:
Sorry, one more, since we discussed it in the context of the user agent: I think we also use kebab case for feature names in most places (not sure we are consistent about it here). Can we do that here?

Reply:
Sure! It is the convention, I just figured out.